packages/api/src/templates/sendEmailTemplate.js

```diff
@@ -21,8 +21,19 @@ const path = require('path');
 const jsdom = require('jsdom');
 const { JSDOM } = jsdom;
 
+const emails = {
+  INVITATION: { subject: '', path: 'invitation_to_farm_email.html' },
+  CONFIRMATION: { subject: '', path: 'send_confirmation_email.html' },
+  WITHHELD_CONSENT: { subject: '', path: 'withheld_consent_email.html' },
+  ACCESS_RESTORE: { subject: '', path: 'restoration_of_access_to_farm_email.html' },
+  ACCESS_REVOKE: { subject: '', path: 'revocation_of_access_to_farm_email.html' },
```

Comment:
Add subject lines

Reply:
This belongs to my next PR, so nvm this for now.